base/multidimensional.jl

```diff
@@ -1176,9 +1176,9 @@ copyto!(::AbstractArray, ::CartesianIndices, ::AbstractArray, ::CartesianIndices
 # circshift!
 circshift!(dest::AbstractArray, src, ::Tuple{}) = copyto!(dest, src)
 """
-    circshift!(dest, src, shifts)
+    circshift!(dest::AbstractArray, src, shifts)
 
-Circularly shift, i.e. rotate, the data in `src`, storing the result in
+Circularly shift, i.e. rotate, the data in array `src`, storing the result in
```

Comment:
Is it it clear that `src` must be an array at this point? If so, why don't we strengthen the signature, too?

Or just play it safe, e.g.
```suggestion
Circularly shift, i.e. rotate, the data in collection `src`, storing the result in
```
or omit this
```suggestion
Circularly shift, i.e. rotate, the data in `src`, storing the result in
```